src/main/java/mil/dds/anet/utils/FkDataLoaderKey.java

```diff
@@ -3,17 +3,19 @@
 // Foreign key batching data loaders
 public enum FkDataLoaderKey {
   APPROVAL_STEP_APPROVERS, // approvalStep.approvers
+  ATTACHMENT_RELATED_OBJECTS, // attachmentRelatedObject.attachments
```

Comment:
Probably rename this to `ATTACHMENT_RELATED_OBJECT_ATTACHMENTS` (in line with notes).